### PR TITLE
Atum updated to v3.3.0

### PR DESCRIPTION
**Pull request by dk1844**
Atum version increased to 3.3.0, notable changes related to that:
 - since Atum v3.3.0 is modularized, it is sufficient to use `atum-model` in `migrations`, `utils` and `menas`.
 - `ControlUtils`'s serialization functionality has been moved to `atum-model` as `SerializationUtils` (thus, the usage now does not require full atum, just the lightweight model is sufficient
 - minor updates

## Naive Test run
 - I have test-ran this version on EMR (std + conf), _INFO file appeared and correctly contains information about being created by Atum 3.3.0

### Release note suggestion:
Atum has been updated to v3.3.0 (modularized version, hence smaller footprint in some submodules)

### CODE REVIEW

**Review by Zejnilovic — APPROVED** [review]

**Review by AdrianOlosutean — APPROVED** [review]

File: menas/pom.xml [hunk 1]
```diff
@@ -265,17 +265,6 @@
             <artifactId>guava</artifactId>
             <version>${guava.version}</version>
         </dependency>
-        <dependency>
```
Comment (benedeki) [inline menas/pom.xml:268]: Why to remove the dependency altogether and not just the exclusion? The Atum is used within Menas, as the files bellow show. This way the dependency is transitive via _Utils_

File: spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala [hunk 2]
```diff
@@ -22,7 +22,7 @@ import org.apache.hadoop.fs.FileSystem
 import org.apache.spark.sql.functions.{lit, to_date}
 import org.apache.spark.sql.{DataFrame, SparkSession}
 import za.co.absa.atum.AtumImplicits
```
Comment (benedeki) [inline spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala:24]: Not needed. Just need to remove `AtumImplicits.SparkSessionWrapper`  on line 117 and 122

**Review by benedeki — CHANGES_REQUESTED** [review]
Please change the PR to be against `develop`?

File: spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala (hunk 2 above)
Comment (dk1844) [reply spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/ConformanceExecution.scala]: Thanks for noticing, done.

File: menas/pom.xml (hunk 1 above)
Comment (dk1844) [reply menas/pom.xml]: Thanks for the explanation, it is indeed cleaner as suggested. Done.

**Review by benedeki — DISMISSED** [review]
- [x] code reviewed
- [ ] pulled
- [ ] built
- [ ] run

File: spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/interpreter/stages/SchemaCheckerSuite.scala [hunk 3]
```diff
@@ -10,6 +10,5 @@ class SchemaCheckerSuite extends AnyFunSuite with SparkTestBase {
     val sourceFile = FileReader.readFileAsString("src/test/resources/data/bug.json")
     val schema = DataType.fromJson(sourceFile).asInstanceOf[StructType]
     val output = SchemaChecker.validateSchemaAndLog(schema)
```
Comment (benedeki) [inline spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/interpreter/stages/SchemaCheckerSuite.scala:12]:
```suggestion
    val output = SchemaChecker.validateSchemaAndLog(schema)
    val expected = (
        List(
          "Validation error for column 'Conformed_TXN_TIMESTAMP', pattern 'yyyy-MM-ddTHH:mm:ss.SSSX': Illegal pattern character 'T'"
        ),
        List()
      )
    assert(output == expected)    
```

Comment (dk1844) [reply spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/interpreter/stages/SchemaCheckerSuite.scala:12]: Of course; applying suggestion.

**Review by benedeki — APPROVED** [review]
Thanks for the good boy scout attitude regarding the `println`s 👍

**Review by Zejnilovic — APPROVED** [review]
I might be weird but I really like how it came to be

Looks good

- [x] code reviewed
- [x] pulled
- [x] built
- [x] run

**Pull request merged**